packages/fx-core/src/plugins/solution/fx-solution/constants.ts

```diff
@@ -168,3 +169,17 @@ export enum SolutionTelemetrySuccess {
 }
 
 export const SolutionTelemetryComponentName = "solution";
+
+export class Messages {
+  // Progress bar messages
+  static readonly DeployArmTemplatesProgressTitle = "Deploying Arm Templates";
```

Comment:
Suggest: it's better to place constant strings in `fx-core/resource/strings.json` file

Reply:
Moved.